.github/workflows/kind-e2e.yaml

```diff
@@ -17,14 +17,14 @@ jobs:
       KO_DOCKER_REPO: kind.local
 
     steps:
-    - name: Set up Go 1.16.x
+    - name: Set up Go 1.17.x
       uses: actions/setup-go@v2
       with:
-        go-version: 1.16.x
+        go-version: 1.17.x
 
     - name: Install Dependencies
       env:
-        KO_VERSION: 0.8.3
+        KO_VERSION: 0.9.3
```

Comment:
Nice, good to synchronize the version across all uses of ko
nit: keep changes atomic and split the `go` bump and `ko` bump

Reply:
shall we use the ko action instead of curling?

Reply:
sounds cool! kk let me try this in the next pr